src/Styles/Scorecard.scss

```diff
@@ -0,0 +1,24 @@
+@import './_variables';
+@import './_mixins';
+
+.player-score-container {
+  @include white-container();
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+.next-hole-btn,
+.end-round-btn {
+  @include button-style();
+}
+
+.hole-number {
+  text-align: center;
+  font-size: 50px;
+  font-weight: 500;
+}
+
```

Comment:
This page looks great. We'll want to do a little bit of aligning, etc. but this is a great start.